### PR TITLE
chore: add telemetry for sdk version

### PR DESCRIPTION
**Pull request by yiqing-zhao**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/16605049
E2E TEST: N/A

### CODE REVIEW

**Review by HuihuiWu-Microsoft — DISMISSED** [review]

File: packages/vscode-extension/src/debug/prerequisitesHandler.ts [hunk 1]
```diff
@@ -1186,6 +1186,16 @@ function checkNpmInstall(
         // treat unexpected error as installed
         error = err;
       }
+
+      const object = await fs.readJson(path.join(folder, "package-lock.json"));
```
Comment (kuojianlu) [inline packages/vscode-extension/src/debug/prerequisitesHandler.ts:1190]: Use better nam instead of `object`.

Comment (swatDong) [inline packages/vscode-extension/src/debug/prerequisitesHandler.ts:1190]: In v3 `npm install` is moved to `teamsfx:deploy` as an action, then this won't be hit.
BTW, does `fs.readJson` throws if *package-lock.json* not exist?

File: packages/vscode-extension/src/debug/prerequisitesHandler.ts [hunk 2]
```diff
@@ -1186,6 +1186,16 @@ function checkNpmInstall(
...
+
+      ctx.properties[TelemetryProperty.BotbuilderVersion] =
+        object?.dependencies["botbuilder"]?.version;
+      ctx.properties[TelemetryProperty.TeamsFxVersion] =
+        object?.dependencies["@microsoft/teamsfx"]?.version;
+      ctx.properties[TelemetryProperty.TeamsJSVersion] =
+        object?.dependencies["@microsoft/teams-js"]?.version;
```
Comment (kuojianlu) [inline packages/vscode-extension/src/debug/prerequisitesHandler.ts:1197]: Are these telemetries also targeted for v3? Please note that npm install cheking is removed in v3.

File: packages/vscode-extension/src/debug/prerequisitesHandler.ts (hunk 1 above)
Comment (yiqing-zhao) [reply packages/vscode-extension/src/debug/prerequisitesHandler.ts]: Thanks for the comments. Updated for not-existed `package-lock.json`, `fs.readJson` will now return `undefined`, and will not affect the telemetry assignment.

Comment (yiqing-zhao) [reply packages/vscode-extension/src/debug/prerequisitesHandler.ts]: Updated, thanks!

File: packages/vscode-extension/src/debug/prerequisitesHandler.ts (hunk 2 above)
Comment (yiqing-zhao) [reply packages/vscode-extension/src/debug/prerequisitesHandler.ts]: Thanks for the comments! Adding check for V3, and the telemetry is currently only for v2. How to add V3 telemetry is under discussion.

**Review by swatDong — APPROVED** [review]

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Pull request merged**